dist/pythonlibs/testrunner/unittest.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2018-19 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import unittest
+from testrunner import setup_child, teardown_child
+
+
+class PexpectTestCase(unittest.TestCase):
+    TIMEOUT = 10
+    LOGFILE = None
```

Comment:
Not sure in unittest will be able to capture test output without `LOGFILE` to stdout or something, even on failures.
But I cannot really verify without using it.

Reply:
Not sure either.. I usually use the output mainly for debugging, so that's why I deactivated it and didn't really think much about actual unittest integration. Let's tackle this when it actually becomes an issue.